src/test/resources/features/delete_team.feature

```diff
@@ -1,10 +1,9 @@
 Feature: Delete Team
   Background:
     Given I login as "owner"
-    And I Create a "XXXGP" Team with "Description" from Dashboards
-    And I go to home from team page
```

Comment:
Without this step, your TC going to fail. 
we test this with Brayan please ensure that the TC going to pass

Reply:
Done!